spec/fixtures/matryoshka/matryoshka.podspec

```diff
@@ -7,6 +7,7 @@ Pod::Spec.new do |s|
   s.homepage         = "http://httpbin.org/html"
   s.source           = { :git => "http://malyutin.local/matryoshka.git", :tag => s.version.to_s }
   s.license          = 'MIT'
+  s.static_framework = true
```

Comment:
surprised this change didnt break other tests :)

We could potentially provide another spec dedicated to static framework testing but I won't push hard on that.

Reply:
I was going to clone it if other test broke, but since it didn't, it might be better to have fewer fixtures.